src/dealDashboard/dealDiscussion/discussionsList/discussionsList.ts

```diff
@@ -59,30 +78,7 @@ export class DiscussionsList{
   private initialize() {
     this.discussionsService.loadDealDiscussions(this.deal.clauseDiscussions);
 
-    this.discussionsArray = Object
-      .keys(this.discussionsService.discussions)
-      .map(key => (
```

Comment:
removed this whole block --> Now use getter (`this.discussions`) directly in the view